TestCases/species_transport/venturi_primitive_3species/species2_primitiveVenturi_mixingmodel_heatcapacity_H2.cfg

```diff
@@ -0,0 +1,152 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Species mixing with 2 species, i.e. 1 transport equations  %
+%                   Including mixture dependent density, viscosity, thermal    %
+%                   conductivity, heat capacities and mass diffusivity         %
+% Author: Cristopher Morales Ubal                                              %
+% Institution: Eindhoven University of Technology                              %
+% Date: 2022/09/12                                                             %
+% File Version 7.4.0 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+SOLVER= INC_RANS
+KIND_TURB_MODEL= SST
+%
+% ---------------- INCOMPRESSIBLE FLOW CONDITION DEFINITION -------------------%
+%
+INC_DENSITY_MODEL= VARIABLE
+INC_DENSITY_INIT= 0.0807
+%
+INC_VELOCITY_INIT= ( 5.00, 0.0, 0.0 )
+%
+INC_ENERGY_EQUATION= YES
+INC_TEMPERATURE_INIT= 300.0
+%
+INC_NONDIM= DIMENSIONAL
+%
+% -------------------- FLUID PROPERTIES ------------------------------------- %
+%
+FLUID_MODEL= FLUID_MIXTURE
+%
+MOLECULAR_WEIGHT= 2.01588, 28.960
+%
+SPECIFIC_HEAT_CP = 14310, 1009.39
+%
```

Comment:
this case represents hydrogen and air mixing? When all properties are consistent with hydrogen and air properties, please mention it in the description of the test case.